proto/viam/app/v1/app.proto

```diff
@@ -624,6 +627,23 @@ message DeleteRobotPartRequest {
   string part_id = 1;
 }
 
+message GetRobotAPIKeysRequest {
+  string robot_id = 1;
+}
+
+message ApiKey {
+  string id = 1;
+  string key = 2;
+  string name = 3;
+  string created_on = 4; 
+  repeated string locations = 5; 
```

Comment:
do we want more information for what resources are using this key apart from a list of names? Could potentially see IDs as helpful if we want to link to a certain page down the line, but that may be too out of scope

also @jr22 if you think this will conflict with your version of ApiKey, I can make some fields optional or change the name.

Reply:
cool thanks! i think for mine ill need similar fields (name, createdOn etc) but I'll also need to include some metadata about the resources (i.e. in that table how it says an api key is on 5 locations and 2 robots for example). not sure if you need that for this api. is this `GetRobotAPIKeys` request primary purpose to be called by the code sample page?

Reply:
yup you are right, i got confused and looked at the wrong designs. Changed this now to be "RobotAPIKey" so we can diff these out. My bad for the confusion sorry !